packages/labs/ssr/package.json

```diff
@@ -33,38 +33,39 @@
   },
   "devDependencies": {
     "@koa/router": "^10.1.1",
-    "@open-wc/testing-karma": "^4.0.9",
     "@open-wc/testing": "^3.0.0-next.1",
+    "@open-wc/testing-karma": "^4.0.9",
     "@types/chai": "^4.2.11",
     "@types/command-line-args": "^5.0.0",
-    "@types/koa__router": "^8.0.2",
+    "@types/koa": "^2.0.49",
     "@types/koa-cors": "*",
     "@types/koa-static": "^4.0.1",
-    "@types/koa": "^2.0.49",
+    "@types/koa__router": "^8.0.2",
     "@types/mocha": "^9.0.0",
-    "@types/node-fetch": "^2.5.7",
     "@types/node": "^16.7.8",
+    "@types/node-fetch": "^2.5.7",
     "@types/parse5": "^6.0.1",
     "@types/resolve": "^1.14.0",
     "@web/test-runner": "^0.13.16",
     "@webcomponents/template-shadowroot": "^0.1.0",
     "chai": "^4.2.0",
     "command-line-args": "^5.1.1",
     "deepmerge": "^4.2.2",
+    "koa": "^2.7.0",
     "koa-cors": "^0.0.16",
     "koa-node-resolve": "^1.0.0-pre.5",
     "koa-static": "^5.0.0",
-    "koa": "^2.7.0",
     "mocha": "^9.1.1",
     "typescript": "^4.3.5",
     "uvu": "^0.5.1"
   },
   "dependencies": {
     "@lit-labs/ssr-client": "^1.0.0",
     "@lit/reactive-element": "^1.1.0",
+    "@types/node": "^17.0.12",
```

Comment:
i'm not sure how much this matters but is node v17 the recommended version for ssr? perhaps we should update the `engines.node` value in `package.json` as well.

Reply:
It doesn't require 17. Might technically work from v12 in global mode, since that's when es modules were added. ES module experimental flag was removed in v13.2.0/v12.17.0. Experimental VM module support was added in v13.0.0/v12.16.0.

I suppose you could make the case that we should be using the oldest support version of the typings? A problem with that, though, is that I'm not sure how well updated the older major versions are with typing bug fixes. I suspect bug-fixes are just added to the latest version.

Reply:
Looking at version history on npm, it does seem like they've done patch updates for even v12 fairly recently. https://www.npmjs.com/package/@types/node/v/12.20.42 was published 10 days ago. If we don't go as far back as 12, still maybe using typings for even/LTS version makes more sense?

Reply:
Active LTS seems reasonable. Switched to 16.